src/api/promptEnhancer.ts

```diff
@@ -0,0 +1,21 @@
+import { Api } from './api'
+import { parseError } from './error'
+
+export type PromptEnhancerArgs = {
+  text: string
+  image: string
+}
+
+async function enhancePrompt({ text, image }: PromptEnhancerArgs): Promise<string> {
```

Comment:
Can we make the enhancePrompt function accept a message as input, and then enhancePrompt will decide whether to enhance or not?

And do not use `!` operator check is there image and is there figmaImageUrl if not just return with some warning `missing ...` for example